daemon/containerd/image_builder.go

```diff
@@ -442,27 +446,37 @@ func (i *ImageService) CreateImage(ctx context.Context, config []byte, parent st
 		})
 	}
 
-	// necessary to prevent the contents from being GC'd
+	createdImage, err := i.createImageOCI(ctx, ociImgToCreate, parentDigest, layers)
```

Comment:
Will this `createImageOCI` be used in more places in future? It looks like it's currently only used here, and the only information consumed from the result is the Digest.


I guess we could return just the digest (or make it return the "clone" as below, but that may require more information to be passed as arguments).

Reply:
(it's not an exported method, so not a blocker)

Reply:
I don't expect it to have more uses. Let me change it to return `image.ID` directly.

Clone is needed only in one case, so I didn't include it in that function.

Reply:
Updated